webapp/portlet/src/main/webapp/activity-composer-app/components/ExoActivityComposer.vue

```diff
@@ -97,7 +103,10 @@ export default {
       attachments: [],
       percent: 100,
       actionsData: [],
-      actionsEvents: []
+      actionsEvents: [],
+      enableSwitchAnArticle: false,
+      showSwitchAnArticle: false,
```

Comment:
There should not be any reference to News addon in social IMO. We should allow to define this behavior by using a plugin from News.